llvm/lib/Target/RISCV/GISel/RISCVLegalizerInfo.cpp

```diff
@@ -707,6 +707,11 @@ RISCVLegalizerInfo::RISCVLegalizerInfo(const RISCVSubtarget &ST)
 bool RISCVLegalizerInfo::legalizeIntrinsic(LegalizerHelper &Helper,
                                            MachineInstr &MI) const {
   Intrinsic::ID IntrinsicID = cast<GIntrinsic>(MI).getIntrinsicID();
+
+  if (RISCVVIntrinsicsTable::getRISCVVIntrinsicInfo(IntrinsicID)) {
+    return true;
+  }
```

Comment:
nit: llvm style prefers simple stmt bodies don't have `{}`s

```suggestion
  if (RISCVVIntrinsicsTable::getRISCVVIntrinsicInfo(IntrinsicID))
    return true;
```

Reply:
Addressed.